### PR TITLE
adapted losses from keras-repo

### PR DESCRIPTION
**Pull request by bfialkoff**
I recently made the change from the keras-repo and found that the pytorch repo didn't have some of the loss functions I liked so I decided to add them

### CODE REVIEW

**Comment by bfialkoff** [comment]
Feeling silly, I see they're already here
